fnet/utils/figures.py

```diff
@@ -213,6 +213,9 @@ def time_series_to_img(im_path_list, window_position = None, window_size = None,
         for im_channel in im_t:
             im = imread(im_channel)
             
+            if len(im.shape) == 4:
+                im = im[:,32,:,:]
+            
             if im.shape[1] == 1:
```

Comment:
i'm not sure this line makes sense anymore

Reply:
@gregjohnso Could you make the changes to `figures.py` and `utils.py`  a separate PR?